tests/integration_tests/test_botorch.py

```diff
@@ -480,3 +488,56 @@ def constraints_func(trial: FrozenTrial) -> Sequence[float]:
     sampler = BoTorchSampler(constraints_func=constraints_func, n_startup_trials=1)
     study = optuna.create_study(sampler=sampler)
     study.optimize(objective, n_trials=3)
+
+
+@pytest.mark.parametrize(
+    "candidates_func, n_objectives",
+    [
+        (integration.botorch.qei_candidates_func, 1),
+        (integration.botorch.qehvi_candidates_func, 2),
+        (integration.botorch.qparego_candidates_func, 4),
+        (integration.botorch.qnehvi_candidates_func, 2),
+        (integration.botorch.qnehvi_candidates_func, 3),  # alpha > 0
+    ],
+)
+def test_botorch_consider_running_trials(candidates_func: Any, n_objectives: int) -> None:
+    sampler = BoTorchSampler(
+        candidates_func=candidates_func,
+        n_startup_trials=1,
+        consider_running_trials=True,
+    )
```

Comment:
An issue with this test is that it alone takes about 75 sec in total, whereas all the other tests seem to take about 60 sec in total. Thus, I'd appreciate suggestions for trimming cases or different ways of testing.

Reply:
Hmm.. Why this test take so long time? 

By the way, we does not execute the integration tests per PR. The integration tests are only executed as scheduled tests. See [here](https://github.com/optuna/optuna/blob/master/.github/workflows/tests-integration.yml#L121) for more details. So, I think we can accept this type of long test.

Reply:
OK, I will leave it as it is. 

As for the reason for the slowness, I do not know for sure but this is possibly because this test has a large number of actual BoTorch optimization for the suggested hyperparameters compared to other tests (5 test parameters * 8 iterations is relatively large).